affected_purls: []
aliases:
- CVE-2015-2317
fixed_purl:
- pkg:deb/debian/python-django@1:1.11.29-1~deb10u1?distro=sid
- pkg:deb/debian/python-django@1.7.7-1?distro=sid
- pkg:deb/debian/python-django@2:2.2.28-1~deb11u1?distro=sid
- pkg:deb/debian/python-django@3:3.2.19-1?distro=sid
references:
- id: 125762
  reference_id: '780873'
  url: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=780873
severities: []
summary: 'The utils.http.is_safe_url function in Django before 1.4.20, 1.5.x, 1.6.x
  before 1.6.11, 1.7.x before 1.7.7, and 1.8.x before 1.8c1 does not properly validate
  URLs, which allows remote attackers to conduct cross-site scripting (XSS) attacks
  via a control character in a URL, as demonstrated by a \x08javascript: URL.'
vulnerability_id: VCID-5rn9-p23r-aaaf
weaknesses: []
